src/log_file.rs

```diff
@@ -14,6 +14,9 @@ use crate::Result;
 /// Text inserted in log files to make important sections more visible.
 pub const LOG_MARKER: &str = "***";
 
+/// Max file name's length for ecrypt fs. Name growths to 255 during encryption
```

Comment:
I realize ecryptfs is the trigger but it's not really an ecryptfs-specific feature. Let's just call it "maximum file name length" and mention ecryptfs as an example.